Implemented solution

kuinorymes opened the pull request:
![Знімок екрана 2025-02-27 183123](https://github.com/user-attachments/assets/980589a6-0715-48b7-bc21-0f3f046d8b46)
![Знімок екрана 2025-02-27 183140](https://github.com/user-attachments/assets/b22f5cd2-adaf-4065-8dd4-1ae2626e6957)
![Знімок екрана 2025-02-27 183146](https://github.com/user-attachments/assets/437e8c67-3b74-4dbb-988f-67fa3375251a)
![Знімок екрана 2025-02-27 183159](https://github.com/user-attachments/assets/a8cf797c-b36d-4ddf-80d1-bba4fee3c117)
![Знімок екрана 2025-02-27 183219](https://github.com/user-attachments/assets/15450647-627a-444e-845a-f8e140884526)
![Знімок екрана 2025-02-27 183235](https://github.com/user-attachments/assets/8211b8b8-ce4e-451a-b074-1517dd19bc51)
![Знімок екрана 2025-02-27 183247](https://github.com/user-attachments/assets/42b16e93-76bd-4265-be8b-2757d9d5f794)
![Знімок екрана 2025-02-27 183359](https://github.com/user-attachments/assets/0c82406c-5ef7-4d76-ad9f-8843f0430ad0)
![Знімок екрана 2025-02-27 183528](https://github.com/user-attachments/assets/398197f4-0152-4f0b-830b-50039e1a33f1)
![Знімок екрана 2025-02-27 183535](https://github.com/user-attachments/assets/1e0b1ffb-69a5-4e02-8087-80321326bee1)
![Знімок екрана 2025-02-27 183549](https://github.com/user-attachments/assets/48c0bba1-c5d8-455a-9e8b-f6eca2f1af0e)
![Знімок екрана 2025-02-27 183554](https://github.com/user-attachments/assets/0e130d20-b1cb-4146-8ab6-8c8c91cdc4c0)
![Знімок екрана 2025-02-27 183609](https://github.com/user-attachments/assets/9bd6381c-a6a8-4a61-8029-4e1e2ffe7b05)
![Знімок екрана 2025-03-08 125931](https://github.com/user-attachments/assets/de1dade4-55ef-4cb2-9c44-11c760ac8c6c)
![Знімок екрана 2025-03-08 125844](https://github.com/user-attachments/assets/209ce889-0595-4a44-b635-9fa8aae6543f)
![Знімок екрана 2025-03-08 125850](https://github.com/user-attachments/assets/6096171d-ca96-48e2-96e9-0723a27157db)

kuinorymes committed:
Implemented solution

Y-Havryliv commented on backend/cinema_service/settings.py:
```diff
@@ -175,3 +177,8 @@
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:5173",
+    "http://127.0.0.1:5173",
+]
```
add blank line to the end of the file

Y-Havryliv commented on frontend/src/views/MovieAddScreen.vue:
```diff
@@ -121,7 +121,7 @@ export default {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies-${movie.id}-upload-image/`, data, imageConfig);
```
did you test this post?

Y-Havryliv commented on frontend/src/views/SignIn.vue:
```diff
@@ -32,7 +32,8 @@ export default {
     async signIn () {
       try {
-        const { data } = await this.axios.post(`${import.meta.env.VITE_API_URL}/api/user/token`, {
+        console.log(`${import.meta.env.VITE_API_URL}/api/user/token/`);
```
delete this log

kuinorymes committed:
Solution fix:
- Added blank line to setting.py
- Deleted console log in SignIn.vue
- Fixed upload-image endpoint in MovieAddScreen.vue
(screenshot in comment)

kuinorymes commented:
@Y-Havryliv Thanks for the feedback
Uploaded additional screenshots(3 last)

Y-Havryliv reviewed (approved):
gj
